js_modules/dagster-ui/packages/ui-core/src/instance/__tests__/DaemonList.test.tsx

```diff
@@ -91,7 +95,13 @@ describe('DaemonList', () => {
     };
 
     render(
-      <MockedProvider mocks={[autoMaterializePausedMock(false), setAutoMaterializePausedMock]}>
+      <MockedProvider
+        mocks={[
+          autoMaterializePausedMock(false),
+          setAutoMaterializePausedMock,
+          autoMaterializePausedMock(true),
```

Comment:
why did we have to add this line?

Reply:
This just cleans up a query warning, since the mutation leads to the status query firing again, but we weren't mocking it.

Reply:
ah gotcha